content/rdi/installation/pacemaker.md

```diff
@@ -282,6 +282,12 @@ To launch the Debezium Server container use the podman service of Pacemaker (Pod
 sudo pcs resource create dbz_server ocf:heartbeat:podman image=docker.io/debezium/server allow_pull=yes run_opts="-v $PWD/conf:/debezium/conf"
 ```
 
+If you are using the Docker resource agent, you should also specify some logging options:
```

Comment:
Can we add a link to the logging documentation for full reference of that options?

Reply:
Do you mean [this](https://github.com/RedisLabs/redislabs-docs/blob/RDSC-1204-document-log-rotations-and-configuration-on-the-pacemaker-page/content/rdi/installation/debezium-server-deployment.md#containerized-deployment)? Or the Docker documentation [here](https://docs.docker.com/config/containers/logging/configure/)?

Reply:
I think the Docker documentation is preferred as it's more comprehensive and covers everything.

Reply:
Done